src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/LogsPage/index.html.twig

```diff
@@ -59,6 +59,8 @@
                   {{ form_widget(logsByEmailForm.logs_by_email) }}
                 </div>
               </div>
+
+              {{ form_rest(logsByEmailForm) }}
```

Comment:
why no block like `webservice_key_form_rest`?
It's a good idea to add a block here

Reply:
I think it does not change anything, does it? :thinking:

Reply:
yes it is, if we want to add custom html & css only for the rest of the form, or anything else.

Reply:
We can modify template if we want to do so, right?

Reply:
So, why adding block everywhere? :trollface: 
Yes we can edit template, but if we want to edit only one part, playing with block is really helpfull, more when the more is really huge and you don't want to break the compatibility in case of refactoring.

Reply:
> we want to edit only one part
Do you mean for a module or in the core ? Because I think it might be impossible for modules right now 😢

Not sure, please confirm @mickaelandrieu

Reply:
It's possible, already did it. With a module you can change the twig block content :)

Reply:
Hmmm, can you drop a module example? :thinking:

Reply:
https://devdocs.prestashop.com/1.7/modules/concepts/templating/admin-views/

Reply:
It overrides page completely, try extending actual block. I don't think it would work.

Reply:
Nope you can extends only a little block, this is exactly what we are doing with `stylesheet`, and `javascript` blocks, and in the example we're overriding `content` block :)
Morewe can also play with twig `embed` tag

Reply:
There is no way to extend exiting template unfortunately. I had to copy/paste all content from original template and modify it instead. Every time you try to extend PS template, it will resolve to modules template resulting in infinite loop I think.

See example https://github.com/Invertus/ps-training/blob/master/views/PrestaShop/Admin/Configure/ShopParameters/preferences.html.twig

Reply:
Wrong :trollface: 
Try to override in your module:
`modules/foo/views/PrestaShop/Admin/Product/CatalogPage/catalog.html.twig`

with 
```
{% extends 'PrestaShopBundle:Admin/Product/CatalogPage:catalog.html.twig' %}

{% block product_catalog_filters %}
  Hello world!
{% endblock %}
```

![image](https://user-images.githubusercontent.com/1462701/58083492-611d7f00-7bb9-11e9-89c7-1cab2c88045e.png)


This is why block are important :)

Reply:
Cool, it does work! I'll add blocks then. :rocket: 

But here's a reason why I though it does not work:

```twig
{# Works well #}
{% extends 'PrestaShopBundle:Admin/Product/CatalogPage:catalog.html.twig' %}
{# Throws "Circular reference" error #}
{% extends '@PrestaShop/Admin/Product/CatalogPage/catalog.html.twig' %}
```

I think this is something we need to fix or document.

Reply:
Because of `@PrestaShop` which is an alias. It checks for all modules templates, if it found a template, it tries to override.

Reply:
Yeah, that's the confusing part.